lib/modules/manager/regex/strategies.ts

```diff
@@ -13,8 +13,8 @@ export function handleAny(
   packageFile: string,
   config: CustomExtractConfig
 ): PackageDependency[] {
-  return config.matchStrings
-    .map((matchString) => regEx(matchString, 'g'))
+  return config
+    .matchStrings!.map((matchString) => regEx(matchString, 'g'))
```

Comment:
please add an override interface to the regex manager where strict required fields are mandatory and validated on initial manager call.

maybe we can also add validation to config validation ( if not already done) and cast to required interface on extractPackageFile call